region/norfair/west/Crocomire Escape.json

```diff
@@ -139,8 +151,44 @@
         "leaveShinecharged": {
           "framesRemaining": 15
         }
+      }
+    },
+    {
+      "link": [2, 2],
+      "name": "Croc Escape Frozen Geruta Shinecharge (Shinespark Out Right)",
+      "reusableRoomwideNotable": "Croc Escape Frozen Geruta Shinecharge",
+      "notable": true,
+      "entranceCondition": {
+        "comeInNormally": {}
+      },
+      "requires": [
+        "h_canTrickyFrozenEnemyRunway",
+        "canCarefulJump",
+        {"enemyDamage": {
```

Comment:
This hit is avoidable at this level with another freeze.
Morph sometimes works but isn't perfect.